--- a/include_menu_array.php	2009-09-12 16:15:13.000000000 -0500
+++ b/include_menu_array.php	2009-09-12 16:16:32.000000000 -0500
@@ -195,6 +195,19 @@ $menue_array = array(
                                     */
                               ),
               ),
+      "55" => array("name"=>"Audits",
+                    "link"=>"#",
+                    "class"=>"menuparent",
+                    "childs"=>array("10"=>array("name"=>"Manage Audits", "link"=>"./audit_manage.php", "image"=>"images/software.png", "title"=>"",),
+                                    "20"=>array("name"=>"Schedule Audit Logs", "link"=>"./list.php?view=audit_logs", "image"=>"images/notes.png", "title"=>"",),
+                                    "30"=>array("name"=>"Run Now Audit Logs", "link"=>"./list.php?view=run_now_logs", "image"=>"images/notes.png", "title"=>"",),
+                                    "40"=>array("name"=>"Perl Cron Log", "link"=>"./list.php?view=cron_log", "image"=>"images/notes.png", "title"=>"",),
+                                    "50"=>array("name"=>"Add Audit Schedule", "link"=>"./audit_schedule.php", "image"=>"images/sched_task_l.png", "title"=>"",),
+                                    "60"=>array("name"=>"Add Audit Configuration", "link"=>"./audit_configuration.php", "image"=>"images/o_specialized.png", "title"=>"",),
+                                    "70"=>array("name"=>"Edit Audit Commands", "link"=>"./audit_commands.php", "image"=>"images/o_specialized.png", "title"=>"",),
+                                    "80"=>array("name"=>"Edit Cron Daemon Settings", "link"=>"./audit_cron_settings.php", "image"=>"images/o_specialized.png", "title"=>"",),
+                              ),
+              ),
       "60" => array("name"=>"Admin",
                     "link"=>"#",
                     "class"=>"menuparent",
