--- a/include_functions.php	2009-09-12 22:13:51.000000000 -0500
+++ b/include_functions.php	2009-09-12 16:26:27.000000000 -0500
@@ -110,6 +110,10 @@
 return $timestamp;
 }
 
+function return_unix_date_time($timestamp) {
+  return date('d/m/y h:i:s a', $timestamp);
+}
+
 function adjustdate($years=0,$months=0,$days=0)
 {
   $todayyear=date('Y');
@@ -224,6 +228,10 @@
 		case "system_last_boot":
 		case "log_timestamp";
 			return return_date_time($myrow[$field["name"]]);
+		case "audit_log_timestamp":
+		case "audit_log_time":
+		case "cron_log_timestamp";
+			return return_unix_date_time($myrow[$field["name"]]);
 		case "system_system_type":
 			if($page=="list") {return determine_img($myrow["system_os_name"],$myrow[$field["name"]]);
 			} else {return $myrow[$field["name"]];}
