--- a/search.php	2009-09-17 21:59:37.000000000 -0500
+++ b/search.php	2009-09-17 22:00:45.000000000 -0500
@@ -33,7 +33,7 @@
 echo "  <td><b>&nbsp;".__("Result")."</b></td>\n";
 echo "</tr>";
 
-$search = stripslashes($_POST["search_field"]);
+$search = (isset($_GET["search"])) ? stripslashes($_GET["search"]) : stripslashes($_POST["search_field"]);
 $search = mysql_real_escape_string($search);
 $search = strtoupper($search);
 
